tests/unittests/tests-uri_parser/tests-uri_parser.c

```diff
@@ -21,10 +21,10 @@
 #include "unittests-constants.h"
 #include "tests-uri_parser.h"
 
-#define VEC(u, f, s, us, h, v6a, z, po, pa, q, e)                           \
+#define VEC(u, f, s, us, h, v6a, z, po_str, po, pa, q, e)                   \
     { .uri = u, .full_uri = f, .scheme = s, .userinfo = us, .host = h,      \
-      .ipv6addr = v6a, .zoneid = z, .port = po, .path = pa,                 \
-      .query = q, .expected = e}
+      .ipv6addr = v6a, .zoneid = z, .port_str = po_str, .port = po,         \
+      .path = pa, .query = q, .expected = e }
 
 #define VEC_CHECK(comp, i, vec_msg)                                         \
```

Comment:
This macro does not compile now, as `port` is not a string anymore. See e.g. https://ci.riot-os.org/RIOT-OS/RIOT/18096/d3bcc56badf6d08d44ba5ef49a2cc3f46167456d/output/compile/tests/unittests/native:gnu.txt